examples.md

```diff
@@ -0,0 +1,26 @@
+# SonarQube examples
+
+## Requirements
+
+ * Docker Engine 1.9
+ * Docker Compose 1.6
+
+## Run SonarQube with Postgres
+
+Go to [this directory](example-compose-files/sq-with-postgres) and run both SonarQube and PostgreSQL containers using [docker-compose](https://github.com/docker/compose):
+
+```bash
+$ docker-compose up
+```
+
+Restart SonarQube container (for example after upgrading or installing a plugin):
+
+```bash
+$ docker-compose restart sonarqube
+```
+
+Analyze a project:
+
+```bash
+mvn sonar:sonar
```

Comment:
We could point to docs where other scanners usage are explained as some people could use different build tool or even language